### DISCUSSION TITLE
How to apply own style for different resolutions

### DISCUSSION THREAD

**Discussion by DenisTukhvatshin** [AvaloniaUI/Avalonia#8447] [General]
I have one view, and it can load it on any computers, but since I use a grid for the layout, everything starts to vary a lot. Somewhere I want to change the text size depending on the screen size or change the ratio(difinitions) in the grid. Please tell me how to solve this problem? make several views for different resolutions? or is it better to do it in one view. Is it all implemented by styles?

**Comment by timunie** [comment]
See https://github.com/AvaloniaUI/Avalonia/discussions/6130

**Reply by maxkatz6** [reply]
This. But also, in the future we want to support @media queries or adaptive triggers.

**Reply by DenisTukhvatshin** [reply]
> See #6130

thanks, this is good information!
